utils/codegen/codegen-py/asn1CodeGenerationUtils.py

```diff
@@ -551,14 +599,18 @@ def asn1TypeToJinjaContext(asn1_type_name: str, asn1_type_info: Dict, asn1_types
             "is_choice_var": True
         })
 
+        if "identified_by" in asn1_type_info:
+            name = asn1_type_info["identified_by"]
+            context["members"][0]["custom_identifier"] = True
```

Comment:
I don't understand why 0 here is working.

Reply:
`context["members"]` is a list of dicts, but holds only one dict.